qadatta.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "Dattatrya More",
+  "type": "personal",
+  "bio": "QA, Test Automation -( UI,API,Performance)",
+  "links": [
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/DattatryaLMore",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/dattatrya-more-94b57513/",
+      "icon": "FaLinkedIn"
```

Please replace it with the value  **' FaLinkedin '**  :+1: 
```suggestion
      "icon": "FaLinkedin"
```